<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>preferences</class>
 <widget class="QDialog" name="preferences">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>538</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configure â€“ QeoDart</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Plain</enum>
         </property>
         <property name="lineWidth">
          <number>0</number>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>438</width>
            <height>446</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <widget class="QGroupBox" name="groupBox">
             <property name="title">
              <string>Language</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="label">
                  <property name="text">
                   <string>Language:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="cobLanguage">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>English</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>The names of the places are available in different languages. Please specify which langauge you prefer:</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lePreferedQcfLanguage">
                <property name="enabled">
                 <bool>true</bool>
                </property>
                <property name="text">
                 <string>ui,en,default</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;You may use comma separated, two letter language codes according to the alpha-2 code of &lt;a href=&quot;http://www.loc.gov/standards/iso639-2/php/code_list.php&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0057ae;&quot;&gt;ISO 639-1&lt;/span&gt;&lt;/a&gt;; in addition, you can use &amp;quot;ui&amp;quot; for the languagen chosen above, and &amp;quot;default&amp;quot; for the name in the native language.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
                <property name="wordWrap">
                 <bool>true</bool>
                </property>
                <property name="openExternalLinks">
                 <bool>true</bool>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_5">
             <property name="title">
              <string>Gameplay</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_5">
                <item>
                 <widget class="QLabel" name="label_14">
                  <property name="text">
                   <string>Ask for</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spbMaxPlaceCount">
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>99</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="label_15">
                  <property name="text">
                   <string>places</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_4">
                <item>
                 <widget class="QLabel" name="label_11">
                  <property name="text">
                   <string>When playing against time, wait at most</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spbMaxTime">
                  <property name="suffix">
                   <string>s</string>
                  </property>
                  <property name="minimum">
                   <number>1</number>
                  </property>
                  <property name="maximum">
                   <number>100</number>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QCheckBox" name="cbResetCursor">
                  <property name="whatsThis">
                   <string>When you are playing with more than one player, you do not want the other players to see where you have clicked. Now you can either move the cursor to another position yourself, or you enable this feature which automatically moves the curosr.</string>
                  </property>
                  <property name="text">
                   <string>Reset cursor when playing with more than one player</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <widget class="QCheckBox" name="cbAutoNewGame">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>Start a new game automatically after showing the results</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="groupBox_2">
             <property name="title">
              <string>Scores and Marks</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <layout class="QGridLayout" name="gridLayout_2">
                <item row="0" column="0">
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>When calculating the score, accepet</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QComboBox" name="cobScoreAreaMode">
                  <property name="whatsThis">
                   <string>When calculating the scores, QeoDart respects the whole area of the place. This area is discribed by a rectangle put around it. As the area of this rectangle often includes areas of other places, the default is accepting only the inner 25% of the rectangle.</string>
                  </property>
                  <item>
                   <property name="text">
                    <string>the point only</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>25% of the area</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>100% of the area</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_12">
                  <property name="text">
                   <string>Use the following grade system:</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
                 <widget class="QComboBox" name="cobGradeSystem">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>German (best: 1.0 - worst: 6.0)</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Do not display marks</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item row="2" column="0">
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Letters per Second:</string>
                  </property>
                 </widget>
                </item>
                <item row="2" column="1">
                 <widget class="QSpinBox" name="spbLettersPerSecond">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>44</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="whatsThis">
                   <string>In the mode &quot;Ask for: Name of Place&quot; in combination with &quot;Against Time&quot;, you get some &quot;free seconds&quot; depending on the length of the name and the number of letters you can type per second.</string>
                  </property>
                  <property name="minimum">
                   <number>2</number>
                  </property>
                  <property name="maximum">
                   <number>10</number>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <spacer name="verticalSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>0</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Delays</string>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>between distance circles</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>before next player</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>before next place</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>before next place (training)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="spbDelayNextCircle">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="spbDelayNextPlayer">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QSpinBox" name="spbDelayNextPlace">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QSpinBox" name="spbDelayNextPlaceTraining">
            <property name="suffix">
             <string>ms</string>
            </property>
            <property name="maximum">
             <number>5000</number>
            </property>
            <property name="singleStep">
             <number>100</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Warnings</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Miscellaneous</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QCheckBox" name="checkBox_2">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="whatsThis">
             <string>Enable this option if QeoDart doesn't recognize the positions of your clicks properly.</string>
            </property>
            <property name="text">
             <string>Use non-default cursor image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblStatusText">
     <property name="text">
      <string notr="true">&lt;statustext&gt;</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok|QDialogButtonBox::Reset|QDialogButtonBox::RestoreDefaults</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>preferences</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>preferences</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
